meta/change_management.md

```diff
@@ -0,0 +1,65 @@
+# Change Management Process
+
+This document describes the change management process for the rule book and all related content and meta files written in Markdown in this project. The process will change based on what phase of the initial project we're in at the moment. Each phase is described below.
+
+Note that changes to this document are also governed by the process described in this document.
+
+## Phase 1: Version 1 Development
+
+The initial version development of this project consists of all work conducted on or after Sunday, November 1, 2020 through to the beginning of phase 2, as defined by phase 2. There are 4 developers involved in phase 1:
+
+- Gryphon Shafer
+- Scott Peterson
+- Jeremy Swingle
+- Zachary Tinker
+
+During the initial version development of this project, all changes will be made as follows:
+
+1. A change's author should submit the change as a *Pull Request* (PR) to the project's `master` branch
+2. At least 1 developer other than the author should comment on the PR; however, ideally many developers will comment
+3. Discussion, debate, and revisions to the PR happen all within GitHub's PR system
+4. The PR is merged to `master` when any of the following is true:
```

Comment:
Do you want to define the process for "resolving comment threads" as well?

Reply:
I agree that the same as merging is great. +1 if in assent with author. +2 if in dissent with author.